src/App.jsx

```diff
@@ -9,32 +9,45 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
```

Comment:
don't need to hardcode data
```suggestion
  const [selectedTabId, setSelectedTabId] = useState(tabs[0].title);
```